perfkitbenchmarker/providers/gcp/gcp_managed_relational_db.py

```diff
@@ -126,6 +126,10 @@ def _Create(self):
     if self.spec.high_availability:
       ha_flag = '--failover-replica-name=replica-' + self.instance_id
       cmd_string.append(ha_flag)
+    if self.spec.backup_enabled:
+      cmd_string.append('--backup')
```

Comment:
`--backup` is enabled by default so if this is not set do you need to use `--no-backup`